lumigator/python/mzai/jobs/eval_lite/eval_lite_config.py

```diff
@@ -0,0 +1,19 @@
+from pydantic import BaseModel, ConfigDict
+
+
+class DatasetConfig(BaseModel):
+    path: str
+    model_config = ConfigDict(extra="forbid")
+
+class JobConfig(BaseModel):
+    max_samples: int = 0
+    storage_path: str
+    input_field: str = "prediction"
```

Comment:
I love that you provide that as a parameter!
Shall we have one for gt too, so we do not assume anything from the input dataset?